src/libraries/System.Net.Http.WinHttpHandler/tests/UnitTests/System.Net.Http.WinHttpHandler.Unit.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <StringResourcesPath>../../src/Resources/Strings.resx</StringResourcesPath>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT</TargetFrameworks>
     <DefineConstants>$(DefineConstants);WINHTTPHANDLER_DLL</DefineConstants>
+    <Nullable>annotations</Nullable>
```

Comment:
Is this required if annotated files explicitly have a `#nullable` directive specified?

Reply:
No not required in that case, but as we will not annotate `test` folders/projects at all adding `#nullable` for them doesn't make sense for me